src/Tickets/Commerce/Admin/Singular_Order_Notices.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * Singular order status notices.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use Tribe__Admin__Notices;
+
+/**
+ * Class Singular_Order_Notices
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+class Singular_Order_Notices {
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_SUCCESSFULLY_UPDATED = 'updated';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_REFUNDED = 'status-refunded';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_SUCCESSFULLY_REFUNDED = 'refunded';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_PARTIALLY_REFUNDED = 'partial-refunded';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_CHANGE_FAILED = 'change-failed';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_REFUND_FAILED = 'refund-failed';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_GATEWAY_REFUND_FAILED = 'gateway-refund-failed';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_GATEWAY_PARTIAL_REFUND_FAILED = 'gateway-partial-refund-failed';
+
+	/**
+	 * List of error notices for the admin.
+	 *
+	 * @since TBD
+	 *
+	 * @var string[]
+	 */
+	protected array $error_messages = [
+		self::ORDER_GATEWAY_PARTIAL_REFUND_FAILED,
+		self::ORDER_REFUND_FAILED,
+		self::ORDER_GATEWAY_REFUND_FAILED,
+		self::ORDER_STATUS_CHANGE_FAILED,
+	];
+
+	/**
+	 * List of successful admin notices.
+	 *
+	 * @since TBD
+	 *
+	 * @var string[]
+	 */
+	protected array $success_messages = [
+		self::ORDER_STATUS_REFUNDED,
+		self::ORDER_STATUS_SUCCESSFULLY_UPDATED,
+		self::ORDER_SUCCESSFULLY_REFUNDED,
+		self::ORDER_PARTIALLY_REFUNDED,
+	];
```

Comment:
They seem like they could be const as well. Not important though

Reply:
Yeah, I'm on the fence only because we may want a filter to add/remove from other plugins so keeping this mutable might be ideal.